### PR TITLE
Added 1.5 Auto

### PR DESCRIPTION
**Pull request by Panda8188**
* Added 1.5 auto
* Added path for 0.5 auto

image
image
image

### CODE REVIEW

File: src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                DriveCal.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ
+            ));
+
+    private HashMap<String, Command> eventMap = new HashMap<>();
+    /* To-do: put events into event map  */
+    public AutoScoreOneFive(DriveSubsystem drive, Arm arm, Grabber grabber, Lights lights) {
+        addCommands(
+            new AutoScoreOne(false, arm, grabber, lights),
+            new FollowPathWithEvents(drive.followTrajectoryCommand(oneFive, true), oneFive.getMarkers(), eventMap)
+                .withTimeout(2.56),
```
Comment (balghane) [inline src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java:37]: Where is this timeout coming from or what is it for

File: src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            new InstantCommand(() -> grabber.intake()),
+            new WaitUntilCommand(()->grabber.seeGamePiece()),
+            new InstantCommand(() -> grabber.spinMotors(GrabberCalibrations.HOLD_GAME_OBJECT_POWER))
```
Comment (balghane) [inline src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java:40]: We have an intake sequence you should be able to use, also you should intake while driving

File: src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java (hunk 1 above)
Comment (Panda8188) [reply src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java:37]: It is the runtime for this path in pathplanner as seen in the image attached above

Comment (Panda8188) [reply src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java]: Realized it is unncessary and subsequently deleted it

File: src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+public class AutoScoreOneFive extends SequentialCommandGroup {
+  private PathPlannerTrajectory oneFive =
+      PathPlanner.loadPath(
+          "OneFive",
+          new PathConstraints(
+              DriveCal.MAX_LINEAR_SPEED_METERS_PER_SEC,
+              DriveCal.MAX_LINEAR_ACCELERATION_METERS_PER_SEC_SQ));
+
+  private HashMap<String, Command> eventMap = new HashMap<>();
+  /* To-do: put events into event map  */
```
Comment (balghane) [inline src/main/java/frc/robot/commands/autos/AutoScoreOneFive.java:27]: Remove todo since you did it

**Review by tails618 — APPROVED** [review]
Lgtm

**Pull request merged**